libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -655,7 +655,7 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 			}
 			else
 				// TODO: Can we bubble up here? There might be different reasons for failure, I think.
-				m_context.appendConditionalRevert(true);
+				m_context.appendConditionalRevert(true, "Uncaught exception in contract creation");
```

Comment:
We might want to append the original reason here, or just bubble up the reason...

Reply:
Sorry, forget that.